sources/packages/forms/src/form-definitions/studentaccountactivity.json

```diff
@@ -357,7 +357,7 @@
                       "properties": {},
                       "allowMultipleMasks": false,
                       "addons": [],
-                      "tag": "p",
+                      "tag": "li",
```

Comment:
This one didn't have any issue with the rendering but I changed to make it a valid HTML.